libpsautohint/src/flat.c

```diff
@@ -178,7 +178,8 @@ FMiniFltn(Cd f0, Cd f1, Cd f2, Cd f3, FltnRec* pfr)
         }
         if (inbbox) {
             Fixed eqa, eqb, x, y;
-            Fixed EPS, d;
+            Fixed EPS;
+            int64_t d;
```

Comment:
Might be good to add a comment like "needs to be int64_t instead of Fixed to prevent overflow" here.

Reply:
(Sorry for the late suggestion, OK with me to leave as is.)

Reply:
Done.

Reply:
👍